backend/code_review_backend/issues/models.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import uuid
+
+from django.db import models
+
+LEVEL_WARNING = "warning"
+LEVEL_ERROR = "error"
+ISSUE_LEVELS = ((LEVEL_WARNING, "Warning"), (LEVEL_ERROR, "Error"))
+
+
+class PhabricatorModel(models.Model):
+    id = models.PositiveIntegerField(primary_key=True)
+    phid = models.CharField(max_length=40, unique=True)
+
+    created = models.DateTimeField(auto_now_add=True)
+    updated = models.DateTimeField(auto_now=True)
+
+    class Meta:
+        abstract = True
+        ordering = ("id",)
+
+
+class Repository(PhabricatorModel):
+
+    slug = models.SlugField(unique=True)
+    url = models.URLField(unique=True)
+    try_url = models.URLField(null=True, blank=True)
```

Comment:
Do we need the `try_url`?

Reply:
I needed it at first to retrieve the file contents on the backend, as the revision is stored on the try project at that stage.
It's not needed now that the hash is built in the bot...